package.json

```diff
@@ -12,7 +12,7 @@
     "build": "esbuild source/index.ts --target=esnext --outfile=dist/ftrack-ts-schema-generator.js --platform=node --format=esm --bundle --packages=external",
     "watch": "esbuild source/index.ts --target=esnext --outfile=dist/ftrack-ts-schema-generator.js --platform=node --format=esm --bundle --packages=external --watch",
     "prepack": "yarn build",
-    "test": "tsc && eslint --cache --fix --max-warnings=0 ./source && prettier -c ./source && vitest",
+    "test": "tsc && eslint --cache --fix --max-warnings=0 ./source && vitest",
```

Comment:
since we are running this test script in our ci, we would like to keep the prettier check here. if you want to run the tests as part of the development, i would recommend just running `yarn vitest` and have it in watch mode instead.

Reply:
Fair enough - that's your choice. I find it strange though to not just have a separate "lint" script that is then invoked in CI. In my opinion, tests and linting have nothing to do with each other.

I re-added it :heart:

Reply:
@jimmycallin did you see that I fixed the review comment? :heart:

Reply:
hi! yes, sorry, we are very short on people during the summer 😭 i'll review it as soon as i can